crates/backend/src/ast.rs

```diff
@@ -36,7 +36,9 @@ pub struct Export {
     pub constructor: Option<String>,
     /// The rust function
     pub function: Function,
+    ///
     pub comments: Vec<String>,
+    /// The name of the rust object the function belongs to TODO is this correct?
```

Comment:
This is the name of the rust function/method, not the object. Check out `impl ToTokens for ast::Export` to see how it is used.